lib/widget/helpers/controllers.dart

```diff
@@ -223,6 +225,7 @@ abstract class WidgetController extends Controller with HasStyles {
     return {
       'flexMode': (value) => flexMode = FlexMode.values.from(value),
       'flex': (value) => flex = Utils.optionalInt(value, min: 1),
+      'automationId': (value) => automationId = Utils.optionalString(value),
```

Comment:
please add a getter as well, it's weird to have a setter but not a getter, we have had many issues due to this asymmetry